portfolio/src/main/webapp/about.css

```diff
@@ -0,0 +1,42 @@
+.font {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.link {
+    color: white;
+} :hover {
```

Comment:
I like that you're adjusting styles on hover! This rule can be put on its own line though.

If you want this to only apply to hovered elements with the `link` class, I think you'll need the selector to read `.link:hover`. There are somewhat nicer ways of handling this sort of thing with scss.

Reply:
fixed! I thought that putting it on the same line like that achieved that, but don't think that works after more reading.